src/views/pantallaRegis/pantallaRegis.js

```diff
@@ -1,33 +1,110 @@
-import { View, Text, Button } from 'react-native'
-import React from 'react'
+import { View, Text, Button, ScrollView } from 'react-native'
+import React, { useState } from 'react'
 import { CajaTexto } from '../../components/cajaTexto/cajaTexto'
 import { CajaTextoVisible } from '../../components/cajaTextoVisible/cajaTextoVisible'
 import styles from './styles'
 import { Check } from '../../components/check/check'
 import { BotonSUP } from '../../components/botonSUP/botonSUP'
 import { BotonSUG } from '../../components/BotonSUG/botonSUG'
 
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  GoogleAuthProvider,
+  signInWithPopup,
+} from 'firebase/auth'
+import { app } from '../../../firebaseconfig'
+
+const auth = getAuth(app)
+//proveedor de gugul
+const provider = new GoogleAuthProvider()
+
 export default function PantallaRegis() {
+  const [user, setUser] = useState({
+    firstName: '',
+    email: '',
+    password: '',
+  })
+
+  //funcion crear cuenta
+  const singUp = () => {
+    createUserWithEmailAndPassword(auth, user.email, user.password)
+      .then((userCredential) => {
+        // Signed in
+        const user = userCredential.user
+        console.log(user)
+        // ...
+      })
+      .catch((error) => {
+        const errorCode = error.code
+        const errorMessage = error.message
+        console.log(errorCode, errorMessage)
```

Comment:
On GitHub, you should not leave debugging code.

Instead you can use an Alert.alert(), indicating some message like: "User not logged in"